scania-digital-design-system/tegel
# test(popover-menu): Adding unit tests

**Describe pull-request**  
Adds unit tests for popover menu.

**Solving issue**  
Fixes: [CDEP-3008](https://tegel.atlassian.net/browse/CDEP-3008)

**How to test**
1. Checkout code
2. Check that `npm run test` works and completes all tests

[CDEP-3008]: https://tegel.atlassian.net/browse/CDEP-3008?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
